src/coreclr/jit/abi.h

```diff
@@ -274,6 +293,8 @@ typedef Arm32Classifier PlatformClassifier;
 typedef RiscV64Classifier PlatformClassifier;
 #elif defined(TARGET_LOONGARCH64)
 typedef LoongArch64Classifier PlatformClassifier;
+#elif defined(TARGET_WASM32)
```

Comment:
```suggestion
#elif defined(TARGET_WASM)
```
I think we'll have pretty much the same code for WASM64 too.